### PR TITLE
daheinze-oscheer/dependency injection

### LINKED ISSUE #29: [daheinze-oscheer] Maps
Complete the Maps lesson

### LINKED ISSUE #33: [daheinze-oscheer] Dependency Injection
Complete the Dependency Injection lesson

### PR DESCRIPTION
**Pull request by oliverscheer**
will close #29 

I ignored the comments ... sorry for that ;-)

### CODE REVIEW

File: daheinze-oscheer/di/di_test.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestGree(t *testing.T) {
```
Comment (dariuszparys) [inline daheinze-oscheer/di/di_test.go]: effective reuse of letters, just genius

**Review by dariuszparys — APPROVED** [review]

File: daheinze-oscheer/di/di.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+)
+
```
Comment (starlord-daniel) [inline daheinze-oscheer/di/di.go:8]: Every exported function should have documentation

**Review by starlord-daniel — APPROVED** [review]
👍

File: daheinze-oscheer/di/di_test.go (hunk 1 above)
Comment (oliverscheer) [reply daheinze-oscheer/di/di_test.go]: LOL :-)

**Pull request merged**